File: TRNMNT/ClientApp/app/core/model/event.model.ts
import { CategoryModel } from "./category.model"


export class EventModel{
    constructor() {
        this.categories = [];
    }

    public eventId: AAGUID;
    public title: string;
    public description: string;
    public address: string;
    public image: any;
    public eventDate: Date;
    public registrationStartTS: Date;
    public registrationEndTS: Date;
    public statusId: number;
    public urlPrefix: string;
    public descritpion : string;
    public tncFilePath: string;
    public cardNumber: string;
    public contactEmail : string;
    public contactPhone : string;
    public fbLink : string;
    public vkLink : string;
    public additionalData: string;
    public categories: CategoryModel[]

};

File: TRNMNT.Core/Model/EventModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TRNMNT.Core.Model
{
    public class EventModel
    {
        public Guid EventId { get; set; }
        public DateTime EventDate { get; set; }
        public DateTime RegistrationStartTS { get; set; }
        public DateTime RegistrationEndTS { get; set; }
        public string ImgPath { get; set; }
        public string Title { get; set; }
        public string UrlPrefix { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public string TNCFilePath { get; set; }
        public string CardNumber { get; set; }
        public string ContactEmail { get; set; }
        public string ContactPhone { get; set; }
        public string FBLink { get; set; }
        public string VKLink { get; set; }
        public string AdditionalData { get; set; }

        public virtual ICollection<CategoryModel> Categories { get; set; }
        
    }
}
